tests/integration/data_sources_and_expectations/data_sources/test_postgresql.py

```diff
@@ -0,0 +1,132 @@
+from datetime import datetime, timezone
+
+import pandas as pd
+import pytest
+
+import great_expectations.expectations as gxe
+from great_expectations.compatibility.sqlalchemy import sqltypes
+from tests.integration.test_utils.data_source_config import PostgreSQLDatasourceTestConfig
+from tests.integration.test_utils.data_source_config.postgres import PostgresBatchTestSetup
+
+pytestmark = pytest.mark.postgresql
+
+
+class TestPostgresqlDataTypes:
+    """This set of tests ensures that we can run expectations against every data
+    type supported by Postgres.
+
+    https://www.postgresql.org/docs/current/datatype.html
+    """
+
+    BOOL_COL_NAME = "my_bool"
+    DATE_COL_NAME = "my_date"
+    NUMERIC_COL_NAME = "my_number"
+    STRING_COL_NAME = "my_string"
+
+    DATA_FRAME = pd.DataFrame(
+        {
+            BOOL_COL_NAME: [True, False, True, False],
+            DATE_COL_NAME: [
+                datetime(2021, 1, 1, tzinfo=timezone.utc).date(),
+                datetime(2021, 1, 2, tzinfo=timezone.utc).date(),
+                datetime(2021, 1, 3, tzinfo=timezone.utc).date(),
+                datetime(2021, 1, 4, tzinfo=timezone.utc).date(),
+            ],
+            NUMERIC_COL_NAME: [1, 2, 3, 4],
+            STRING_COL_NAME: ["a", "b", "c", "d"],
+        }
+    )
+
+    def test_boolean(self):
+        batch_setup = PostgresBatchTestSetup(
+            config=PostgreSQLDatasourceTestConfig(
+                column_types={self.BOOL_COL_NAME: sqltypes.BOOLEAN}
+            ),
+            data=self.DATA_FRAME,
+            extra_data={},
```

Comment:
i do wish `extra_data` wasn't a required param but no need to change here

Reply:
agreed.